Readme.md

```diff
@@ -118,15 +118,17 @@ This boilerplate uses `typechain` capabilties to prevent errors and ensure corre
 
 Boilerplate comes with some example tasks:
 
-- Get balance of an address: `npx hardhat balance-of --tokenaddress 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512 --account 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --network localhost`
+- Get balance of an address: `npx hardhat balance-of --tokenaddress 0x5FbDB2315678afecb367f032d93F642f64180aa3 --account 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --network localhost`
```

Comment:
NIT: I would say we won't modify the previous examples unless some arguments of the method change. It's not a big deal but you'll see some changes and probably someone can get confused.